unblob/handlers/filesystem/squashfs.py

```diff
@@ -61,7 +61,7 @@ class SquashFSv3Handler(_SquashFSBase):
             uint32 block_size;
             uint32 fragments;
             uint32 fragment_table_start_2;
-            int64  bytes_used;
+            uint64  bytes_used;
```

Comment:
We may not use the other fields but aren't they supposed to be the same types?

Reply:
They are supposed to be the same type. Fixed.